### ISSUE TITLE
New Rule: `no-replace-test-comments`

### ISSUE THREAD

**Issue by jaydgruber** [ember-cli/eslint-plugin-ember#717]
### What
- disallow 'Replace this with your real tests' comments

### Why
- we should encourage folks to use ember-cli + generators
- for those in a rush / without strong reviews, you end up with plenty of checked in test files that only have canary tests:
```javascript
  // Replace this with your real tests.
  test('it exists', function (assert) {
    let service = this.owner.lookup('service:company');
    assert.ok(service);
  });
```
- this rule will nudge developers to start writing a meaningful test earlier in the process

**Comment by jaydgruber** [comment]
Just ran this rule on my repo from work
`✖ 21 problems (21 errors, 0 warnings)`
oh dear 😞

**Comment by jaydgruber** [comment]
As another issue, plan to introduce a fixer that prefixes TODO:
https://github.com/ember-cli/eslint-plugin-ember/pull/718#pullrequestreview-377235724

**Comment by bmish** [comment]
I actually don't think we should have an autofixer for this. I would prefer to just update the Ember blueprint to add the `// TODO:` prefix in future generated comments.

**Comment by jaydgruber** [comment]
Yeah, I had the same thought a bit later. I'll probably still do it for fun as an exercise. Haven't learned how fixers work yet.

@trabus, i know you've done some blueprint stuff. Can you point me towards the place to update a blueprint so I can stampede off thataway?

**Comment by jakebixbyavalara** [comment]
Any of the folders here that have `-test` appended:
https://github.com/emberjs/ember.js/tree/master/blueprints

Ember and Ember-Data include their own blueprints so we can change them with each respective version. The files contained in the various `-files` folders are the lodash templates used to generate the tests:
https://github.com/emberjs/ember.js/blob/master/blueprints/controller-test/qunit-files/__root__/__testType__/__path__/__test__.js#L8

**Comment by jaydgruber** [comment]
opened https://github.com/emberjs/ember.js/pull/18828
